ui/src/components/Inventory/InventoryFormModal.js

```diff
@@ -0,0 +1,158 @@
+import Button from '@material-ui/core/Button'
+import Checkbox from '@material-ui/core/Checkbox'
+import Dialog from '@material-ui/core/Dialog'
+import DialogActions from '@material-ui/core/DialogActions'
+import DialogContent from '@material-ui/core/DialogContent'
+import DialogTitle from '@material-ui/core/DialogTitle'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import Grid from '@material-ui/core/Grid'
+import MenuItem from '@material-ui/core/MenuItem'
+import React from 'react'
+import TextField from '../Form/TextField'
+import { Field, Form, Formik } from 'formik'
+
+const validateRequired = value => (value || typeof value === 'number' ? undefined : 'Required')
+const validatePositiveNumber = value => 
```

Comment:
Optional check: A null value is can be entered if the 0's are deleted.

Reply:
Implemented a check for an empty string `''` in commit 34e7176e11f9d1cfb470e582f113750362cb1e02